containers/ddev-xhgui/Dockerfile

```diff
@@ -1,4 +1,20 @@
-FROM xhgui/xhgui:0.21 AS ddev-xhgui
+FROM xhgui/xhgui:0.21 AS xhgui-docker
+
+FROM alpine:3.15 AS ddev-xhgui
```

Comment:
you should build `FROM scratch` then if you copy other image all files to avoid superfluous files or even cause conflicts because different os files don't merge well.

Reply:
Thank you.